frontend/src/pages/learn/index.tsx

```diff
@@ -83,30 +94,38 @@ const Learn: FC = () => {
       <Modal
         onClose={handleCloseModal}
         open={isModalOpen}
-        className="mx-auto flex w-full items-center justify-center border-none bg-transparent p-1 backdrop:bg-black backdrop:bg-opacity-80 md:w-2/3 lg:w-2/5"
+        className="mx-auto flex w-full border-none bg-transparent p-1 
+        backdrop:bg-black backdrop:bg-opacity-80 backdrop-blur-sm"
       >
-        <FocusOn enabled={isModalOpen} className="w-full">
+        <FocusOn enabled={isModalOpen} className="w-full md:py-16">
           <section
             data-autofocus
             tabIndex={-1}
-            className="rounded-md bg-white p-6"
+            className="rounded-md bg-white mx-auto items-center justify-center 
+            p-6 h-full sm:w-full md:w-2/3 lg:w-3/5 xl:w-2/5 overflow-y-auto "
             aria-describedby={`QuizModal-header`}
           >
-            <div className="flex justify-end gap-2 p-2">
-              <Button
-                variant="text"
-                className="text-base font-bold normal-case text-primary-700 hover:bg-white"
-                onClick={handleCloseModal}
-              >
-                <CloseIcon className="mr-2 inline text-2xl font-bold text-primary-700" />{' '}
-                {t('quiz.navigation.close')}
-              </Button>
-            </div>
-            <div className="rounded-3xl bg-[#f5f5f5] font-display">
-              <h2 className="mb-14 p-10 text-left text-5xl font-bold text-primary-700">
-                {t('quiz.navigation.title')}
-              </h2>
-            </div>
+            <div className='flex flex-col h-full'>
+            {!showConfirmation ? (
```

Comment:
This can be replaced with `{!showConfirmation && /* element */}`

Reply:
Issue solved